lib/mfs.js

```diff
@@ -0,0 +1,125 @@
+var stream = require('stream')
+
+module.exports = function (node) {
+  var store = {}
+  var baseDir = '/npm-registry/'
```

Comment:
Should this be configurable and passed in, or does the baseDir not really matter too much and be basically be any string?